lib/intersection_validator.rb

```diff
@@ -0,0 +1 @@
+IntersectionValidator = Class.new(ActiveModelValidatesIntersectionOf::Validator)
```

Comment:
There is some differences in the loading order:
http://www.virtuouscode.com/2008/08/26/classnew-and-inherited/

And the second way is easily readable, don't agree?